### PR TITLE
Dynamic cleanup+example (WIP)

### PR DESCRIPTION
**Pull request by BunkerCoder**
If this looks correct, I'll start on testing (new example looking good).  TIA for the review.

### CODE REVIEW

File: src/IQFeed.CSharpApiClient/Streaming/Level1/Level1DynamicFields.cs [hunk 1]
```diff
@@ -223,7 +225,7 @@ public class Level1DynamicFields
 
         public static Level1DynamicFields Parse(string message, DynamicFieldset[] fields)
         {
-            var values = message.SplitFeedMessage();
+            var values = message.SplitFeedMessage().Skip(1).ToArray();
```
Comment (mathpaquette) [inline src/IQFeed.CSharpApiClient/Streaming/Level1/Level1DynamicFields.cs:228]: whats for?

File: src/IQFeed.CSharpApiClient/Streaming/Level1/Messages/UpdateSummaryDynamicMessage.cs [hunk 2]
```diff
@@ -43,11 +44,6 @@ public override int GetHashCode()
             }
         }
 
-        public override string ToString()
```
Comment (mathpaquette) [inline src/IQFeed.CSharpApiClient/Streaming/Level1/Messages/UpdateSummaryDynamicMessage.cs:46]: this is needed

File: src/IQFeed.CSharpApiClient/Streaming/Level1/Messages/UpdateSummaryDynamicMessage.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Text;
```
Comment (mathpaquette) [inline src/IQFeed.CSharpApiClient/Streaming/Level1/Messages/UpdateSummaryDynamicMessage.cs:2]: can remove it.

**Comment by mathpaquette** [comment]
@BunkerCoder look also 1 test is failing. https://ci.appveyor.com/project/mathpaquette/iqfeed-csharpapiclient/builds/33518962#L873

**Comment by mathpaquette** [comment]
Something like this is good enough isn’t?
https://github.com/mathpaquette/IQFeed.CSharpApiClient/commit/ce9d2c300429fef07d0049b0340f21980a922654

**Comment by mathpaquette** [comment]
Instead of doing this I guess you can do:

replace
for (var i = 0; i < fields.Length; i++)

to:
for (var i = 1; i < fields.Length; i++)

Just to avoid creating 2 arrays.
SplitFeedMessage() == create an array
Skip(1) == create another array.

Thanks.

**Comment by BunkerCoder** [comment]
I think this addresses concerns and issues.  If it looks OK, I'll complete the test TODOs.

File: src/IQFeed.CSharpApiClient/Streaming/Level1/Handlers/Level1MessageDynamicHandler.cs [hunk 4]
```diff
@@ -12,6 +12,8 @@ public class Level1MessageDynamicHandler : BaseLevel1MessageHandler, ILevel1Mess
 
         public void SetDynamicFields(params DynamicFieldset[] fieldNames)
         {
+            if (fieldNames[0] != DynamicFieldset.Symbol)
```
Comment (mathpaquette) [inline src/IQFeed.CSharpApiClient/Streaming/Level1/Handlers/Level1MessageDynamicHandler.cs:15]: good

File: src/IQFeed.CSharpApiClient/Streaming/Level1/DynamicFieldset.cs [hunk 5]
```diff
@@ -115,9 +115,6 @@ public enum DynamicFieldset
         [FieldsetDescription("Last Time", typeof(TimeSpan))]
         LastTime,
 
-        [FieldsetDescription("Last Trade Date", typeof(DateTime))]
```
Comment (mathpaquette) [inline src/IQFeed.CSharpApiClient/Streaming/Level1/DynamicFieldset.cs:118]: doesnt exist anymore?

Comment (BunkerCoder) [reply src/IQFeed.CSharpApiClient/Streaming/Level1/DynamicFieldset.cs:118]: Nope. Looks like they changed name to Last Date.

Comment (mathpaquette) [reply src/IQFeed.CSharpApiClient/Streaming/Level1/DynamicFieldset.cs:118]: on the 6.1 or 6.0 docs?

Comment (BunkerCoder) [reply src/IQFeed.CSharpApiClient/Streaming/Level1/DynamicFieldset.cs:118]: Checked http://www.iqfeed.net/dev/api/docs/docs60/Level1UpdateSummaryMessage.cfm and http://www.iqfeed.net/dev/api/docs/Level1UpdateSummaryMessage.cfm.
Both include Last Date.  Neither include Last Trade Date.

Comment (mathpaquette) [reply src/IQFeed.CSharpApiClient/Streaming/Level1/DynamicFieldset.cs:118]: sounds good!

Comment (mathpaquette) [reply src/IQFeed.CSharpApiClient/Streaming/Level1/DynamicFieldset.cs:118]: cool, just add missing tests and we're all set to merge that feature! thank you so much for your patience !

File: src/IQFeed.CSharpApiClient/Streaming/Level1/Messages/UpdateSummaryDynamicMessage.cs (hunk 2 above)
Comment (BunkerCoder) [reply src/IQFeed.CSharpApiClient/Streaming/Level1/Messages/UpdateSummaryDynamicMessage.cs]: fixed...sorry

File: src/IQFeed.CSharpApiClient/Streaming/Level1/Messages/UpdateSummaryDynamicMessage.cs (hunk 3 above)
Comment (BunkerCoder) [reply src/IQFeed.CSharpApiClient/Streaming/Level1/Messages/UpdateSummaryDynamicMessage.cs]: removed

File: src/IQFeed.CSharpApiClient/Streaming/Level1/DynamicFieldset.cs (hunk 5 above)
Comment (BunkerCoder) [reply src/IQFeed.CSharpApiClient/Streaming/Level1/DynamicFieldset.cs:118]: Patience? You've been responding in seconds!  Thanks for laying out the design for me!

**Comment by BunkerCoder** [comment]
Possibly, maybe done.  Thanks again for your help.

**Pull request merged**